verification/verify.go

```diff
@@ -153,3 +160,75 @@ func IsRetryable(err error) bool {
 	_, retryable := err.(Retryable)
 	return retryable
 }
+
+func verifyPixelParams(params *Params) error {
+	// check if the node is on-chain mode
```

Comment:
Is the onchain check mandatory? I never had an issue testing verification in offchain mode.

If we still need the check, it might be good to exit early to save one level of indentation. Like:

```golang
if <offchain> {
  return
}
... rest of the code
```

Reply:
Previously, an off-chain B wouldn't run pixel count verification and since we're moving pixel count verification into `SegmentVerifier` we'll need an on-chain check to preserve that behavior (given the assumption that we still don't want to run pixel count verification for off-chain Bs due to the unnecessary overhead).

>  it might be good to exit early to save one level of indentation

I agree that returning early if the `params.OrchestratorTicketParams == nil` helps with readability.

Reply:
I think what may be missing here is there is no default verification policy? So does this bit of code actually get executed at all?

If If we instantiate a default verification policy only in onchain mode, that would mitigate the need for any on-chain checks here, which feels a bit out of place.

Reply:
> I think what may be missing here is there is no default verification policy? So does this bit of code actually get executed at all?

Yeah you're right - this wouldn't be executed until #1328 (describes an empty policy with a nil verifier) is implemented. Though your point about setting `server.Policy` only when the node is in on-chain mode makes sense - removing the on-chain check is nice.

Reply:
fixed in 10dfcc561387ae04b10531de3abb3dfae4d24b2b for early return when not in on-chain mode